[BUG] ERROR 500 when workflow to update document title is enabled

### Description

Currently I am getting 500 errors when saving one specific file while I have a document title workflow turned on. If it is off i am not getting any errors.

### Steps to reproduce

1. Have Update Title on Document Update Workflow enabled
2. Change document metadata 
3. Save File
4. Get 500 error and no Title Update
5. Close document (ignore popups) 
6. Open document (ignore popups)
7. See metadata changed 

![Image](https://github.com/user-attachments/assets/128b638c-443c-40ad-8e48-4ca9406e667e)
![Image](https://github.com/user-attachments/assets/5967ed98-5b40-4671-86fd-8d53e2257310)

### Webserver logs

```bash
Error: 
{"headers":{"normalizedNames":{},"lazyUpdate":null},"status":500,"statusText":"Internal Server Error","url":"http://umbrel:2349/api/documents/129/","ok":false,"name":"HttpErrorResponse","message":"Http failure response for http://umbrel:2349/api/documents/129/: 500 Internal Server Error","error":"\n<!doctype html>\n<html lang=\"en\">\n<head>\n  <title>Server Error (500)</title>\n</head>\n<body>\n  <h1>Server Error (500)</h1><p></p>\n</body>\n</html>\n"}

Logs: 
[2025-02-10 17:07:15,278] [INFO] [paperless.matching] Document matched WorkflowTrigger 5 from Workflow: Document Title

[2025-02-10 17:07:15,278] [INFO] [paperless.handlers] Applying WorkflowAction 3 from Workflow: Document Title

[2025-02-10 17:12:59,302] [INFO] [paperless.matching] Document did not match Workflow: New Invoices

[2025-02-10 17:12:59,302] [DEBUG] [paperless.matching] ('Document doc type Contracts does not match Invoice',)

[2025-02-10 17:12:59,308] [INFO] [paperless.matching] Document matched WorkflowTrigger 5 from Workflow: Document Title

[2025-02-10 17:12:59,308] [INFO] [paperless.handlers] Applying WorkflowAction 3 from Workflow: Document Title

[2025-02-10 17:18:40,885] [INFO] [paperless.matching] Document did not match Workflow: New Invoices

[2025-02-10 17:18:40,885] [DEBUG] [paperless.matching] ('Document doc type Contracts does not match Invoice',)

[2025-02-10 17:18:40,890] [INFO] [paperless.matching] Document matched WorkflowTrigger 5 from Workflow: Document Title

[2025-02-10 17:18:40,890] [INFO] [paperless.handlers] Applying WorkflowAction 3 from Workflow: Document Title
```

### Browser logs

```bash

```

### Paperless-ngx version

2.14.7

### Host OS

umbrel 1.4-b1 (installed through that 

### Installation method

Other (please describe above)

### System status

```json
{
    "pngx_version": "2.14.7",
    "server_os": "Linux-6.1.0-30-amd64-x86_64-with-glibc2.36",
    "install_type": "docker",
    "storage": {
        "total": 1963188352000,
        "available": 1046822130688
    },
    "database": {
        "type": "postgresql",
        "url": "paperless",
        "status": "OK",
        "error": null,
        "migration_status": {
            "latest_migration": "documents.1061_workflowactionwebhook_as_json",
            "unapplied_migrations": []
        }
    },
    "tasks": {
        "redis_url": "redis://paperless_broker_1:6379",
        "redis_status": "OK",
        "redis_error": null,
        "celery_status": "OK",
        "index_status": "OK",
        "index_last_modified": "2025-02-10T17:18:40Z",
        "index_error": null,
        "classifier_status": "OK",
        "classifier_last_trained": "2025-02-10T17:05:16.705745Z",
        "classifier_error": null
    }
}
```

### Browser

_No response_

### Configuration changes

_No response_

### Please confirm the following

- [x] I believe this issue is a bug that affects all users of Paperless-ngx, not something specific to my installation.
- [x] This issue is not about the OCR or archive creation of a specific file(s). Otherwise, please see above regarding OCR tools.
- [x] I have already searched for relevant existing issues and discussions before opening this report.
- [x] I have updated the title field above with a concise description.

got the time to go finding logs in the container and got ones that show much more now.  most of this doesn't show up on the `/logs` page  

```
paperless_webserver_1  | [2025-02-11 21:26:45,072] [INFO] [celery.worker.strategy] Task documents.tasks.consume_file[01f5d37e-5b56-4c90-b6ce-2cc75f678ca1] received
paperless_webserver_1  | [2025-02-11 21:26:45,155] [INFO] [paperless.matching] Document matched WorkflowTrigger 3 from Workflow: New Invoices
paperless_webserver_1  | [2025-02-11 21:26:45,163] [INFO] [paperless.tasks] WorkflowTriggerPlugin completed with: Applying WorkflowAction 1 from Workflow: New Invoices
paperless_webserver_1  | [2025-02-11 21:26:45,173] [INFO] [paperless.consumer] Consuming 2025-02-10 very long file name part with many words  2025-02-10 - shorter name with less letters but still many.pdf
paperless_webserver_1  | [2025-02-11 21:26:45,231] [INFO] [paperless.parsing.tesseract] pdftotext exited 0
paperless_webserver_1  | [2025-02-11 21:26:45,634] [INFO] [ocrmypdf._pipelines.ocr] Start processing 4 pages concurrently
paperless_webserver_1  | [2025-02-11 21:26:45,636] [INFO] [ocrmypdf._pipeline] skipping all processing on this page
paperless_webserver_1  | [2025-02-11 21:26:45,636] [INFO] [ocrmypdf._pipeline] skipping all processing on this page
paperless_webserver_1  | [2025-02-11 21:26:45,636] [INFO] [ocrmypdf._pipeline] skipping all processing on this page
paperless_webserver_1  | [2025-02-11 21:26:45,636] [INFO] [ocrmypdf._pipeline] skipping all processing on this page
paperless_webserver_1  | [2025-02-11 21:26:45,637] [INFO] [ocrmypdf._pipeline] skipping all processing on this page
paperless_webserver_1  | [2025-02-11 21:26:45,637] [INFO] [ocrmypdf._pipeline] skipping all processing on this page
paperless_webserver_1  | [2025-02-11 21:26:45,654] [INFO] [ocrmypdf._pipelines.ocr] Postprocessing...
paperless_webserver_1  | [2025-02-11 21:26:47,494] [INFO] [ocrmypdf._pipeline] Image optimization ratio: 1.00 savings: 0.0%
paperless_webserver_1  | [2025-02-11 21:26:47,494] [INFO] [ocrmypdf._pipeline] Total file size ratio: 0.97 savings: -3.1%
paperless_webserver_1  | [2025-02-11 21:26:47,499] [INFO] [ocrmypdf._pipelines._common] Output file is a PDF/A-2B (as expected)
paperless_webserver_1  | [2025-02-11 21:26:47,754] [INFO] [paperless.parsing.tesseract] pdftotext exited 0
paperless_webserver_1  | [2025-02-11 21:26:48,697] [INFO] [paperless.parsing] convert exited 0
paperless_webserver_1  | [2025-02-11 21:26:50,461] [INFO] [paperless.handlers] Assigning correspondent long named correspondent to 2025-02-11 2025-02-10 very long file name part with many words  2025-02-10 - shorter name with less letters but still many
paperless_webserver_1  | [2025-02-11 21:26:50,513] [INFO] [paperless.handlers] Assigning document type Invoice to 2025-02-11 long named correspondent 2025-02-10 very long file name part with many words  2025-02-10 - shorter name with less letters but still many
paperless_webserver_1  | [2025-02-11 21:26:50,567] [INFO] [paperless.handlers] Tagging 2025-02-11 long named correspondent 2025-02-10 very long file name part with many words  2025-02-10 - shorter name with less letters but still many with Personal, Healthcare
paperless_webserver_1  | [2025-02-11 21:26:50,776] [INFO] [paperless.matching] Document did not match Workflow: New Invoices
paperless_webserver_1  | [2025-02-11 21:26:50,798] [INFO] [paperless.consumer] Document 2025-02-11 long named correspondent 2025-02-10 very long file name part with many words  2025-02-10 - shorter name with less letters but still many consumption finished
paperless_webserver_1  | [2025-02-11 21:26:50,802] [INFO] [paperless.tasks] ConsumeTaskPlugin completed with: Success. New document id 134 created
paperless_webserver_1  | [2025-02-11 21:26:50,809] [INFO] [celery.app.trace] Task documents.tasks.consume_file[01f5d37e-5b56-4c90-b6ce-2cc75f678ca1] succeeded in 5.733195945969783s: 'Success. New document id 134 created'
paperless_webserver_1  | [2025-02-11 21:26:57,578] [INFO] [paperless.matching] Document did not match Workflow: New Invoices
paperless_webserver_1  | [2025-02-11 21:26:57,583] [INFO] [paperless.matching] Document matched WorkflowTrigger 5 from Workflow: Document Title
paperless_webserver_1  | [2025-02-11 21:26:57,583] [INFO] [paperless.handlers] Applying WorkflowAction 3 from Workflow: Document Title
paperless_webserver_1  | [2025-02-11 21:26:57,597] [ERROR] [django.request] Internal Server Error: /api/documents/134/
paperless_webserver_1  | Traceback (most recent call last):
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/backends/utils.py, line 105, in _execute
paperless_webserver_1  |     return self.cursor.execute(sql, params)
paperless_webserver_1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/psycopg/cursor.py, line 97, in execute
paperless_webserver_1  |     raise ex.with_traceback(None)
paperless_webserver_1  | psycopg.errors.StringDataRightTruncation: value too long for type character varying(128)
paperless_webserver_1  | 
paperless_webserver_1  | The above exception was the direct cause of the following exception:
paperless_webserver_1  | 
paperless_webserver_1  | Traceback (most recent call last):
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/asgiref/sync.py, line 518, in thread_handler
paperless_webserver_1  |     raise exc_info[1]
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/core/handlers/exception.py, line 42, in inner
paperless_webserver_1  |     response = await get_response(request)
paperless_webserver_1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/asgiref/sync.py, line 518, in thread_handler
paperless_webserver_1  |     raise exc_info[1]
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/core/handlers/base.py, line 253, in _get_response_async
paperless_webserver_1  |     response = await wrapped_callback(
paperless_webserver_1  |                ^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/asgiref/sync.py, line 468, in __call__
paperless_webserver_1  |     ret = await asyncio.shield(exec_coro)
paperless_webserver_1  |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/asgiref/current_thread_executor.py, line 40, in run
paperless_webserver_1  |     result = self.fn(*self.args, **self.kwargs)
paperless_webserver_1  |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/asgiref/sync.py, line 522, in thread_handler
paperless_webserver_1  |     return func(*args, **kwargs)
paperless_webserver_1  |            ^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/views/decorators/csrf.py, line 65, in _view_wrapper
paperless_webserver_1  |     return view_func(request, *args, **kwargs)
paperless_webserver_1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/rest_framework/viewsets.py, line 124, in view
paperless_webserver_1  |     return self.dispatch(request, *args, **kwargs)
paperless_webserver_1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/rest_framework/views.py, line 509, in dispatch
paperless_webserver_1  |     response = self.handle_exception(exc)
paperless_webserver_1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/rest_framework/views.py, line 469, in handle_exception
paperless_webserver_1  |     self.raise_uncaught_exception(exc)
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/rest_framework/views.py, line 480, in raise_uncaught_exception
paperless_webserver_1  |     raise exc
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/rest_framework/views.py, line 506, in dispatch
paperless_webserver_1  |     response = handler(request, *args, **kwargs)
paperless_webserver_1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/src/paperless/src/documents/views.py, line 401, in update
paperless_webserver_1  |     document_updated.send(
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/dispatch/dispatcher.py, line 189, in send
paperless_webserver_1  |     response = receiver(signal=self, sender=sender, **named)
paperless_webserver_1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/src/paperless/src/documents/signals/handlers.py, line 563, in run_workflows_updated
paperless_webserver_1  |     run_workflows(
paperless_webserver_1  |   File /usr/src/paperless/src/documents/signals/handlers.py, line 1178, in run_workflows
paperless_webserver_1  |     document.save()
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/models/base.py, line 892, in save
paperless_webserver_1  |     self.save_base(
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/models/base.py, line 998, in save_base
paperless_webserver_1  |     updated = self._save_table(
paperless_webserver_1  |               ^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/models/base.py, line 1130, in _save_table
paperless_webserver_1  |     updated = self._do_update(
paperless_webserver_1  |               ^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/models/base.py, line 1195, in _do_update
paperless_webserver_1  |     return filtered._update(values) > 0
paperless_webserver_1  |            ^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/models/query.py, line 1278, in _update
paperless_webserver_1  |     return query.get_compiler(self.db).execute_sql(CURSOR)
paperless_webserver_1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/models/sql/compiler.py, line 2003, in execute_sql
paperless_webserver_1  |     cursor = super().execute_sql(result_type)
paperless_webserver_1  |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/models/sql/compiler.py, line 1574, in execute_sql
paperless_webserver_1  |     cursor.execute(sql, params)
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/backends/utils.py, line 79, in execute
paperless_webserver_1  |     return self._execute_with_wrappers(
paperless_webserver_1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/backends/utils.py, line 92, in _execute_with_wrappers
paperless_webserver_1  |     return executor(sql, params, many, context)
paperless_webserver_1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/backends/utils.py, line 100, in _execute
paperless_webserver_1  |     with self.db.wrap_database_errors:
paperless_webserver_1  |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/utils.py, line 91, in __exit__
paperless_webserver_1  |     raise dj_exc_value.with_traceback(traceback) from exc_value
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/backends/utils.py, line 105, in _execute
paperless_webserver_1  |     return self.cursor.execute(sql, params)
paperless_webserver_1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/psycopg/cursor.py, line 97, in execute
paperless_webserver_1  |     raise ex.with_traceback(None)
paperless_webserver_1  | django.db.utils.DataError: value too long for type character varying(128)
Download Paperless-ngx logspaperless_webserver_1  | [2025-02-11 21:26:45,072] [INFO] [celery.worker.strategy] Task documents.tasks.consume_file[01f5d37e-5b56-4c90-b6ce-2cc75f678ca1] received
paperless_webserver_1  | [2025-02-11 21:26:45,155] [INFO] [paperless.matching] Document matched WorkflowTrigger 3 from Workflow: New Invoices
paperless_webserver_1  | [2025-02-11 21:26:45,163] [INFO] [paperless.tasks] WorkflowTriggerPlugin completed with: Applying WorkflowAction 1 from Workflow: New Invoices
paperless_webserver_1  | [2025-02-11 21:26:45,173] [INFO] [paperless.consumer] Consuming 2025-02-10 very long file name part with many words  2025-02-10 - shorter name with less letters but still many.pdf
paperless_webserver_1  | [2025-02-11 21:26:45,231] [INFO] [paperless.parsing.tesseract] pdftotext exited 0
paperless_webserver_1  | [2025-02-11 21:26:45,634] [INFO] [ocrmypdf._pipelines.ocr] Start processing 4 pages concurrently
paperless_webserver_1  | [2025-02-11 21:26:45,636] [INFO] [ocrmypdf._pipeline] skipping all processing on this page
paperless_webserver_1  | [2025-02-11 21:26:45,636] [INFO] [ocrmypdf._pipeline] skipping all processing on this page
paperless_webserver_1  | [2025-02-11 21:26:45,636] [INFO] [ocrmypdf._pipeline] skipping all processing on this page
paperless_webserver_1  | [2025-02-11 21:26:45,636] [INFO] [ocrmypdf._pipeline] skipping all processing on this page
paperless_webserver_1  | [2025-02-11 21:26:45,637] [INFO] [ocrmypdf._pipeline] skipping all processing on this page
paperless_webserver_1  | [2025-02-11 21:26:45,637] [INFO] [ocrmypdf._pipeline] skipping all processing on this page
paperless_webserver_1  | [2025-02-11 21:26:45,654] [INFO] [ocrmypdf._pipelines.ocr] Postprocessing...
paperless_webserver_1  | [2025-02-11 21:26:47,494] [INFO] [ocrmypdf._pipeline] Image optimization ratio: 1.00 savings: 0.0%
paperless_webserver_1  | [2025-02-11 21:26:47,494] [INFO] [ocrmypdf._pipeline] Total file size ratio: 0.97 savings: -3.1%
paperless_webserver_1  | [2025-02-11 21:26:47,499] [INFO] [ocrmypdf._pipelines._common] Output file is a PDF/A-2B (as expected)
paperless_webserver_1  | [2025-02-11 21:26:47,754] [INFO] [paperless.parsing.tesseract] pdftotext exited 0
paperless_webserver_1  | [2025-02-11 21:26:48,697] [INFO] [paperless.parsing] convert exited 0
paperless_webserver_1  | [2025-02-11 21:26:50,461] [INFO] [paperless.handlers] Assigning correspondent long named correspondent to 2025-02-11 2025-02-10 very long file name part with many words  2025-02-10 - shorter name with less letters but still many
paperless_webserver_1  | [2025-02-11 21:26:50,513] [INFO] [paperless.handlers] Assigning document type Invoice to 2025-02-11 long named correspondent 2025-02-10 very long file name part with many words  2025-02-10 - shorter name with less letters but still many
paperless_webserver_1  | [2025-02-11 21:26:50,567] [INFO] [paperless.handlers] Tagging 2025-02-11 long named correspondent 2025-02-10 very long file name part with many words  2025-02-10 - shorter name with less letters but still many with Personal, Healthcare
paperless_webserver_1  | [2025-02-11 21:26:50,776] [INFO] [paperless.matching] Document did not match Workflow: New Invoices
paperless_webserver_1  | [2025-02-11 21:26:50,798] [INFO] [paperless.consumer] Document 2025-02-11 long named correspondent 2025-02-10 very long file name part with many words  2025-02-10 - shorter name with less letters but still many consumption finished
paperless_webserver_1  | [2025-02-11 21:26:50,802] [INFO] [paperless.tasks] ConsumeTaskPlugin completed with: Success. New document id 134 created
paperless_webserver_1  | [2025-02-11 21:26:50,809] [INFO] [celery.app.trace] Task documents.tasks.consume_file[01f5d37e-5b56-4c90-b6ce-2cc75f678ca1] succeeded in 5.733195945969783s: 'Success. New document id 134 created'
paperless_webserver_1  | [2025-02-11 21:26:57,578] [INFO] [paperless.matching] Document did not match Workflow: New Invoices
paperless_webserver_1  | [2025-02-11 21:26:57,583] [INFO] [paperless.matching] Document matched WorkflowTrigger 5 from Workflow: Document Title
paperless_webserver_1  | [2025-02-11 21:26:57,583] [INFO] [paperless.handlers] Applying WorkflowAction 3 from Workflow: Document Title
paperless_webserver_1  | [2025-02-11 21:26:57,597] [ERROR] [django.request] Internal Server Error: /api/documents/134/
paperless_webserver_1  | Traceback (most recent call last):
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/backends/utils.py, line 105, in _execute
paperless_webserver_1  |     return self.cursor.execute(sql, params)
paperless_webserver_1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/psycopg/cursor.py, line 97, in execute
paperless_webserver_1  |     raise ex.with_traceback(None)
paperless_webserver_1  | psycopg.errors.StringDataRightTruncation: value too long for type character varying(128)
paperless_webserver_1  | 
paperless_webserver_1  | The above exception was the direct cause of the following exception:
paperless_webserver_1  | 
paperless_webserver_1  | Traceback (most recent call last):
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/asgiref/sync.py, line 518, in thread_handler
paperless_webserver_1  |     raise exc_info[1]
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/core/handlers/exception.py, line 42, in inner
paperless_webserver_1  |     response = await get_response(request)
paperless_webserver_1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/asgiref/sync.py, line 518, in thread_handler
paperless_webserver_1  |     raise exc_info[1]
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/core/handlers/base.py, line 253, in _get_response_async
paperless_webserver_1  |     response = await wrapped_callback(
paperless_webserver_1  |                ^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/asgiref/sync.py, line 468, in __call__
paperless_webserver_1  |     ret = await asyncio.shield(exec_coro)
paperless_webserver_1  |           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/asgiref/current_thread_executor.py, line 40, in run
paperless_webserver_1  |     result = self.fn(*self.args, **self.kwargs)
paperless_webserver_1  |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/asgiref/sync.py, line 522, in thread_handler
paperless_webserver_1  |     return func(*args, **kwargs)
paperless_webserver_1  |            ^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/views/decorators/csrf.py, line 65, in _view_wrapper
paperless_webserver_1  |     return view_func(request, *args, **kwargs)
paperless_webserver_1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/rest_framework/viewsets.py, line 124, in view
paperless_webserver_1  |     return self.dispatch(request, *args, **kwargs)
paperless_webserver_1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/rest_framework/views.py, line 509, in dispatch
paperless_webserver_1  |     response = self.handle_exception(exc)
paperless_webserver_1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/rest_framework/views.py, line 469, in handle_exception
paperless_webserver_1  |     self.raise_uncaught_exception(exc)
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/rest_framework/views.py, line 480, in raise_uncaught_exception
paperless_webserver_1  |     raise exc
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/rest_framework/views.py, line 506, in dispatch
paperless_webserver_1  |     response = handler(request, *args, **kwargs)
paperless_webserver_1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/src/paperless/src/documents/views.py, line 401, in update
paperless_webserver_1  |     document_updated.send(
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/dispatch/dispatcher.py, line 189, in send
paperless_webserver_1  |     response = receiver(signal=self, sender=sender, **named)
paperless_webserver_1  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/src/paperless/src/documents/signals/handlers.py, line 563, in run_workflows_updated
paperless_webserver_1  |     run_workflows(
paperless_webserver_1  |   File /usr/src/paperless/src/documents/signals/handlers.py, line 1178, in run_workflows
paperless_webserver_1  |     document.save()
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/models/base.py, line 892, in save
paperless_webserver_1  |     self.save_base(
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/models/base.py, line 998, in save_base
paperless_webserver_1  |     updated = self._save_table(
paperless_webserver_1  |               ^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/models/base.py, line 1130, in _save_table
paperless_webserver_1  |     updated = self._do_update(
paperless_webserver_1  |               ^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/models/base.py, line 1195, in _do_update
paperless_webserver_1  |     return filtered._update(values) > 0
paperless_webserver_1  |            ^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/models/query.py, line 1278, in _update
paperless_webserver_1  |     return query.get_compiler(self.db).execute_sql(CURSOR)
paperless_webserver_1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/models/sql/compiler.py, line 2003, in execute_sql
paperless_webserver_1  |     cursor = super().execute_sql(result_type)
paperless_webserver_1  |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/models/sql/compiler.py, line 1574, in execute_sql
paperless_webserver_1  |     cursor.execute(sql, params)
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/backends/utils.py, line 79, in execute
paperless_webserver_1  |     return self._execute_with_wrappers(
paperless_webserver_1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/backends/utils.py, line 92, in _execute_with_wrappers
paperless_webserver_1  |     return executor(sql, params, many, context)
paperless_webserver_1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/backends/utils.py, line 100, in _execute
paperless_webserver_1  |     with self.db.wrap_database_errors:
paperless_webserver_1  |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/utils.py, line 91, in __exit__
paperless_webserver_1  |     raise dj_exc_value.with_traceback(traceback) from exc_value
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/django/db/backends/utils.py, line 105, in _execute
paperless_webserver_1  |     return self.cursor.execute(sql, params)
paperless_webserver_1  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
paperless_webserver_1  |   File /usr/local/lib/python3.12/site-packages/psycopg/cursor.py, line 97, in execute
paperless_webserver_1  |     raise ex.with_traceback(None)
paperless_webserver_1  | django.db.utils.DataError: value too long for type character varying(128)
```

--- a/src/documents/signals/handlers.py
+++ b/src/documents/signals/handlers.py
@@ -1178,6 +1178,8 @@ def run_workflows(
                     webhook_action()
 
             if not use_overrides:
+                # limit title to 128 characters
+                document.title = document.title[:128]
                 # save first before setting tags
                 document.save()
                 document.tags.set(doc_tag_ids)

